numpy/fft/__init__.py

```diff
@@ -118,8 +118,16 @@
 It differs from the forward transform by the sign of the exponential
 argument and the default normalization by :math:`1/n`.
 
+Type Promotion
+--------------
+
+`numpy.fft` is inconsistent with other NumPy functions as it promotes
+`float32` and `complex64` arrays to `float64` and `complex128` arrays.
+For an FFT implementation that does not promote input arrays, see `scipy.fftpack`.
+
 Normalization
 -------------
+
```

Comment:
why this change?

Reply:
To make this heading consistent with the surrounding headings. It appears that all the other headings have white space both above and below. This heading only had whitespace above.